src/twix.coffee

```diff
@@ -455,7 +455,7 @@ makeTwix = (moment) ->
   Twix
 
 # -- MAKE AVAILABLE
-return module.exports = makeTwix(require 'moment') if hasModule
+return module.exports = makeTwix(if moment then return moment else require 'moment') if hasModule
```

Comment:
won't this throw if `moment` hasn't been defined as a global?

Reply:
In plain JS will work, in CoffeeScript honestly, I don't know. Just double-checked, this code passes tests `return module.exports = makeTwix(if moment? then moment else require 'moment') if hasModule`

Also, it may have adverse effects on existing users, so technically this should be part of a major "potentially breaking" release.